dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -510,6 +588,9 @@ public CachedContextAndSchemas createJAXBContextAndSchemas(Set<Class<?>> classes
         //add user extra class into jaxb context
         if (extraClass != null && extraClass.length > 0) {
             for (Class<?> clz : extraClass) {
+                if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
```

Comment:
It might be a good idea to use a variable having the value LOG.isLoggable(Level.FINE) here. Because it's a loop, it might have so many repetitions.

Reply:
good point, will update